openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -930,12 +915,15 @@ def label_molecules(self, topology, verbose=False):
            Or should we label all interactions in a :class:`Topology` instead of just labeling its ``unique_molecules``?
 
         """
+        from openforcefield.topology import Topology
         # Loop over molecules and label
         molecule_labels = list()
         for molecule_idx, molecule in enumerate(topology.reference_molecules):
+            top_mol = Topology.from_molecules([molecule])
             current_molecule_labels = dict()
-            for parameter_handler in self._parameter_handlers.values():
-                matches = parameter_handler.find_matches(molecule)
-                molecule_labels[molecule_idx][parameter_handler.name] = matches
+            for tag, parameter_handler in self._parameter_handlers.items():
+                matches = parameter_handler.get_matches(top_mol)
```

Comment:
I think I renamed `get_matches` to `find_matches` in the previous PR.

Reply:
Thanks. Fixed in be96018